python/lsst/display/firefly/footprints.py

```diff
@@ -0,0 +1,67 @@
+#
+# LSST Data Management System
+# Copyright 2018 LSST Corporation.
```

Comment:
Should start using the [new copyright approach](https://developer.lsst.io/stack/license-and-copyright.html) here, especially in new code.